app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.controller.ts

```diff
@@ -7,11 +7,14 @@ import { IArtifact, IExpectedArtifact, NamingService } from '@spinnaker/core';
 import { GitCredentialType, IAppengineAccount } from 'appengine/domain/index';
 import { AppengineSourceType, IAppengineServerGroupCommand } from '../serverGroupCommandBuilder.service';
 
+import { SETTINGS } from '@spinnaker/core';
```

Comment:
Please import this with the other `@spinnaker/core` imports (`IArtifact`, `NamingService`) above.